### PR TITLE
Add Ruby and Rails style guide

### PR DESCRIPTION
**Pull request by lewispb**
Really just putting into words what we already do.

### CODE REVIEW

**Comment by TThurston** [comment]
👍  Good to have this recorded - nice one

File: style/ruby/.rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    - "**/Gemfile"
+    - "**/Rakefile"
+    - "**/bin/*"
+    - "**/config.ru"
+  Exclude:
+    - "config/**/*"
+    - "script/**/*"
+    - !ruby/regexp /old_and_unused\.rb$/
+    # Rubocop is accidentally linting HTML ERB files in Atom Editor https://github.com/AtomLinter/linter-rubocop/issues/48
+    - "**/*.html.erb"
```
Comment (lewispb) [inline style/ruby/.rubocop.yml]: Actually, seems like this is fixed now, so we could remove this?

https://github.com/AtomLinter/linter-rubocop/issues/48

Comment (Cruikshanks) [inline style/ruby/.rubocop.yml]: Yep, seems like a good idea to me. Could you do it as a separate commit (don't squash this branch) though so we have the record of why we removed it somewhere.

**Comment by TThurston** [comment]
@lewispb @Cruikshanks Having read this again I have a couple of additional comments:
- Is the "Tools" section proscriptive enough for a pedant?  What is our rule; do we just run these tools and take their output as advice, or do we run these tools and ensure we have no issues before committing?
- Regarding running a `bundle update` every week... how will we do this in practice?  Have we identified CI tools that can do this, or are we asking a person to undertake this task manually each week?  For all projects or only new ones?

**Comment by Cruikshanks** [comment]
I think what @lewispb has done is a good starter for ten. What I would actually do is move the tools section out and possibly split it. The stuff about **Brakeman** and how we ensure things like it and **Rubocop** get run would go into a new section possibly called *Continuous Integration*. In there I would talk about how new projects should have CI from the start, and what is expected to be run in your CI.

On the bundle update thing I believe we have a solution. [Deppbot](https://www.deppbot.com/) was trialled on WEX, and has been in continuous use on the [Admin Prototype Kit](https://github.com/EnvironmentAgency/admin-prototype-kit) with no issue. It does exactly that. Monitors your repo and when updates our found it automatically generates a pull request on which it performs a bundle update. So straight away you see whether your CI will build following the update. It also works for projects not actively being developed. Check out <https://github.com/EnvironmentAgency/admin-prototype-kit/pull/66> for an example.

So for now I was going to leave this as is, but re-jig it around was I can get that follow up PR done as I'd rather have something than nothing. Does that help?

**Comment by lewispb** [comment]
Sounds good to me @Cruikshanks, I've added that commit regarding html.erb as well now.
